core/main/src/firebolt/handlers/advertising_rpc.rs

```diff
@@ -35,13 +32,9 @@ use ripple_sdk::{
                 AdvertisingFrameworkConfig, AdvertisingRequest, AdvertisingResponse, GetAdConfig,
             },
             fb_capabilities::{CapabilityRole, FireboltCap, RoleInfo},
-            fb_general::{ListenRequest, ListenerResponse},
         },
         gateway::rpc_gateway_api::CallContext,
-        storage_property::{
-            StorageProperty, EVENT_ADVERTISING_POLICY_CHANGED,
-            EVENT_ADVERTISING_SKIP_RESTRICTION_CHANGED,
-        },
```

Comment:
Should we also clean-up all of the references in storage_property.rs as well (`StorageProperty::SkipRestriction`, `PROPERTY_DATA_SKIP_RESTRICTION`, `EVENT_ADVERTISING_*`, etc.)?